pkg/volume/util/io_util.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+//IoUtil is a util for common IO operations
+//it also backports certain operations from golang 1.5
+type IoUtil interface {
```

Comment:
> Can I do a follow-up pull request which can be tested separately?

Yes. Just create a bug to follow up and put a TODO comment with the issue # here saying that this code is duplicate and it should be consolidated with `ioHandler` defined in `pkg/volume/fc/fc_util.go`.